src/config.ts

```diff
@@ -89,6 +95,12 @@ const defaultConfig: IConfig = {
         allowNoPrefix: false,
         additionalPrefixes: [],
     },
+    protections: {
+        wordlist: {
+            words: ["nigger", "faggot", "tranny", "retard"],
```

Comment:
the default should be empty in this case:
```suggestion
            words: [],
```